tests/initializers/linear/sinkhorn_init_test.py

```diff
@@ -229,28 +233,34 @@ def test_gauss_pointcloud_geom(self, rng: jnp.ndarray):
 
   @pytest.mark.parametrize('lse_mode', [True, False])
   @pytest.mark.parametrize("jit", [False, True])
-  @pytest.mark.parametrize("initializer", ["sorting", "gaussian"])
+  @pytest.mark.parametrize("initializer", ["sorting", "gaussian", "subsample"])
   def test_initializer_n_iter(
       self, rng: jnp.ndarray, lse_mode: bool, jit: bool,
-      initializer: Literal["sorting", "gaussian"]
+      initializer: Literal["sorting", "gaussian", "subsample"]
   ):
     """Tests Gaussian initializer"""
     n, m, d = 200, 200, 2
+    subsample_n = 100
     epsilon = 1e-2
 
-    if initializer == "gaussian":
+    if initializer == "sorting":
+      ot_problem = create_sorting_problem(rng, n=n, epsilon=epsilon)
+    else:
       ot_problem = create_ot_problem(
           rng, n, m, d, epsilon=epsilon, batch_size=3
       )
-    else:
-      ot_problem = create_sorting_problem(rng, n=n, epsilon=epsilon)
 
     if jit:
       run_fn = jax.jit(
           run_sinkhorn, static_argnames=["initializer", "lse_mode"]
       )
+      sub_run_fn = jax.jit(
```

Comment:
I think this can be done in a nicer way - consider refactoring the `run_sinkhorn` to accept the initializer instance (instead of a string + kwargs).